student-store-express-api/models/store.js

```diff
@@ -2,7 +2,6 @@ const { storage } = require("../data/storage");
 const { BadRequestError } = require("../utils/errors");
 
 class Store {
```

Comment:
Store could also be called a "Service" and you could have ProductsService, OrderService, etc which would be the layer between the controller and the db